src/human.js

```diff
@@ -16,12 +17,14 @@ export default class Human extends Player {
 
   keyup(e) {
     if (e.key === ' ') {
+      this.interacting = false;
       this.avatar.cancel();
     }
   }
 
   keydown(e) {
-    if (e.key === ' ') {
+    if (e.key === ' ' && !this.interacting) {
+      this.interacting = true;
```

Comment:
You can get rid of `this.interacting` altogether, and switch this to `if (e.key === ' ' && !e.repeat)`

Reply:
thats pretty neat

Reply:
A big thing we're trying to refactor out is not having flag states, and relying more on derived states instead.

Reply:
I did check for a way before making this change, but I didn't look very well